### ISSUE TITLE
'file' disk template cannot be disabled when drbd instances are running

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1039]
Originally reported of Google Code with ID 983.

```
<b>What steps will reproduce the problem?</b>
1. create a cluster with drbd instances
2. try to disable 'file' disk template

<b>What is the expected output? What do you see instead?</b>

Ganeti complains that it cannot disable it, because file instances are running.
```
Originally added on 2014-10-31 12:59:43 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 5b9b2a449f12d2d6eec090a83aba1cbaffcd6f0e
Author: Helga Velroyen <helgav@google.com>
Date:   Wed Nov 5 11:40:03 2014 +0100

    Check only for root-disk templates when disabling templates
    
    When disabling disk-templates on a cluster, a check is made
    whether there are still instances using these templates.
    This check errorneously checked not only the root disk,
    but all child disks as well, which in this context is not
    correct. We fix this by adding a separate function to
    test only root disks.
    
    This fixes issue 983.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Hrvoje Ribicic <riba@google.com>
```
Originally added on 2014-11-05 14:46:44 +0000 UTC.

Mentioned Issue with GoogleCode ID 983 has been migrated to GitHub with ID #1039.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-11-05 14:46:53 +0000 UTC.

Added to Milestone: Release2.10
Removed from Milestone: Release2.9

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:40:20 +0000 UTC.

Changed State: Released
